test/e2e_node/pod_hostnamefqdn_test.go

```diff
@@ -103,13 +111,13 @@ var _ = SIGDescribe("Hostname of Pod [Feature:SetHostnameAsFQDN][NodeAlphaFeatur
 	                hostname command returns shortname (pod name in this case), and hostname -f returns FQDN.
 	*/
 	ginkgo.It("a pod with subdomain field has FQDN, hostname is shortname", func() {
-		pod := testPod()
+		pod := testPod("hostfqdn")
 		pod.Spec.Containers[0].Command = []string{"sh", "-c", "echo $(hostname)';'$(hostname -f)';'"}
 		subdomain := "t"
 		// Set PodSpec subdomain field to generate FQDN for pod
 		pod.Spec.Subdomain = subdomain
 		// Expected Pod FQDN
-		hostFQDN := fmt.Sprintf("%s.%s.%s.svc.%s", pod.ObjectMeta.Name, subdomain, f.Namespace.Name, framework.TestContext.ClusterDNSDomain)
+		hostFQDN := fmt.Sprintf("%s.%s.%s.svc.%s", pod.ObjectMeta.Name, subdomain, f.Namespace.Name, dnsDomain)
 		output := []string{fmt.Sprintf("%s;%s;", pod.ObjectMeta.Name, hostFQDN)}
 		// Create Pod
 		f.TestContainerOutput("shotname and fqdn", pod, 0, output)
```

Comment:
is it `shortname` or `shotname`?

```suggestion
		f.TestContainerOutput("shortname and fqdn", pod, 0, output)
```

Reply:
good catch, thanks